src/foam/core/AbstractDatePropertyInfo.java

```diff
@@ -85,4 +85,43 @@ public void updateSignature(FObject obj, Signature sig) throws SignatureExceptio
     long val = date.getTime();
     sig.update((ByteBuffer) bb.get().putLong(val).flip());
   }
+
+  protected abstract java.util.Date get_(Object o);
+  protected abstract java.util.Date cast(Object key);
+    
+//  public foam.lib.parse.Parser jsonParser() {
+//    return foam.lib.json.DateParser.instance() == null ? foam.lib.json.DateParser.instance(): null;
+//  }
+  
```

Comment:
Is there a sqlType - are you taking them from java/refinements?

Reply:
Yes.

Reply:
Are you going to copy it over?

Reply:
For sqlType, it is now in each specific property info.